doc/usr/tuto/start.md

```diff
@@ -1,431 +1,112 @@
-
-### My first demo scene
-
-Now that you are familiar with the controller, we are going to write our own
-first demo.
-
-Edit a script such as `~/mydemo.py` and add the following:
-
-```python
-import pynopegl as ngl
-from pynopegl_utils.misc import scene
-
-
-_VERTEX = '''
-void main()
-{
-    ngl_out_pos = ngl_projection_matrix * ngl_modelview_matrix * vec4(ngl_position, 1.0);
-    var_tex0_coord = (tex0_coord_matrix * vec4(ngl_uvcoord, 0.0, 1.0)).xy;
-}
-'''
-
-
-_FRAGMENT = '''
-void main()
-{
-    ngl_out_color = ngl_texvideo(tex0, var_tex0_coord);
-}
-'''
-
-
-@scene()
-def test_demo(cfg):
-    geometry = ngl.Quad()
-    media = ngl.Media(cfg.medias[0].filename)
-    texture = ngl.Texture2D(data_src=media)
-    program = ngl.Program(vertex=_VERTEX, fragment=_FRAGMENT)
-    program.update_vert_out_vars(var_tex0_coord=ngl.IOVec2())
-    render = ngl.Render(geometry, program)
-    render.update_frag_resources(tex0=texture)
-    return render
-```
-
-You should be able to preview your scene with `ngl-control -m ~/mydemo.py` and
-observe a centered quadrilateral geometry with the video playing in it. But
-first, let's look at the `Graph view` tab in the controller to understand the
-scene we just crafted:
-
-![my-demo](img/graph-simple-render.png)
-
-To formulate what we observe here, we'll say the following: the `Render` is the
-node orchestrating the render of the `Texture2D` (identified by "*tex0*") in the
-`Quad` geometry, using the `Media` as data source for filling the texture.
-
-### Pimp my demo
-
-**Note**: writing GLSL shaders is out of the scope of this tutorial, so we will
-assume you are comfortable with the basis. If not, you may want to look at [the
-book of shaders][book-of-shaders] or any beginner resource of your choice.
-
-You may want to refer to the [vertex and fragment shader
-parameters][expl-shaders] documentation to know which parameters are exposed by
-`nope.gl`.
-
-We are now going to pimp a little our fragment shader. Instead of just picking
-into the texture, we will mix it with some red by replacing the `ngl_out_color`
-assignment with the following:
-
-```glsl
-void main()
-{
-    vec4 color = vec4(1.0, 0.0, 0.0, 1.0);
-    vec4 video = ngl_texvideo(tex0, var_tex0_coord);
-    ngl_out_color = mix(video, color, 0.5);
-}
+Edit a new script such as `mydemo.py` and add the following code:
 
+```{nope} starter.hello_world
+:export_type: image
+Hello World!
 ```
 
-![my reddish demo](img/ngl-control-reddish-scene.png)
+Loading that script in the viewer should provide the same rendering observed
+here.
 
-[book-of-shaders]: http://thebookofshaders.com/
-[expl-shaders]: /usr/expl/shaders.md
+In this first example all we did was to return a single node (the [Text] node),
+in a `@scene()` decorated function for the viewer to find it. The function
+receives a configuration of type `SceneCfg` which gives us access to various
+metadata on the scene (for both reading and sometimes writing).
```

Comment:
- of the scene ?
- maybe list a few metata ?

Reply:
Done